web/portal/src/main/webapp/WEB-INF/web.xml

```diff
@@ -205,6 +196,11 @@
   </filter-mapping>
 
   <filter-mapping>
+    <filter-name>LocalizationFilter</filter-name>
+    <url-pattern>/*</url-pattern>
```

Comment:
Even the filter will be triggered on all the requests but the compute of the language will be done only when the information not yet persisted in the ConversationState.
In other words, If the language information is already persisted in the ConversationState, the filter will be triggered but it doesn't anything .So, it has not a performance impact.